spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
There is both the Discussion API (used by mobile) as well as the LMS.

I'm not sure if it is worth the effort if we don't know the future of the comment service, but I think the way to remove the default would be to temporarily have no default (at least locally).  
1. Enforce that `with_responses` is always passed explicitly by the caller and fix all calls from Discussion API and LMS.
2. Release Discussion API and LMS changes to Production.
3. Free to change default, because it isn't yet in use.
4. Once default is changed, we can remove explicit passing of the new default value from Discussion API and LMS.